app/src/main/res/values/styles.xml

```diff
@@ -452,6 +452,32 @@
         <item name="cornerSize">20dp</item>
     </style>
 
+    <!-- SearchBar -->
+
+    <style name="SearchTextInputLayoutStyle">
+        <item name="boxStrokeColor">@null</item>
+        <item name="boxStrokeWidth">0dp</item>
+        <item name="endIconContentDescription">@string/contentDescriptionButtonDeleteSearch</item>
+        <item name="endIconDrawable">@drawable/ic_close_search</item>
+        <item name="endIconMode">clear_text</item>
+        <item name="hintEnabled">false</item>
+        <item name="placeholderText">@string/searchFieldPlaceholder</item>
```

Comment:
You took the string from the Search searchbar which says "Search an email" and it is reused in the folder search but in the folder search we search folders and not messages

Reply:
Nice try but you've simply put "search a folder" in the search this time